### PR TITLE
Increase celery worker app instances to 2, move manifest files to new directory

### PR DESCRIPTION
**Pull request by lbeaufort**
## Summary

Issue api#612
  
- Move manifest files to new directory
- Separate celery manifests per space
- increase instances to 2
- clean up deploy task
- modify rollback function to work for both api and celery
- Bind celery to creds service

## How to test

Review deploy outputs

- [x] Test deploy to dev with new directory change: https://app.circleci.com/pipelines/github/fecgov/fecfile-web-api/2819/workflows/adb3dfb6-5fa3-4892-9653-81a7ca1b1c47/jobs/8888
- [x] Test deploy to dev after celery, deploy task changes: https://app.circleci.com/pipelines/github/fecgov/fecfile-web-api/2824/workflows/787c92a4-31d6-4403-8905-9cf52f55308a/jobs/8908

```
type:            web
sidecars:        
instances:       2/2
memory usage:    1024M
start command:   bin/run-celery-worker.sh
     state     since                  cpu    memory         disk           details
#0   running   2023-12-28T21:03:58Z   0.0%   455.1M of 1G   425.4M of 1G   
#1   running   2023-12-28T21:04:08Z   0.0%   0 of 1G        0 of 1G       
```

### CODE REVIEW

File: tasks.py
Comment (toddlees) [inline tasks.py:89]: I think the new directory didn't get added

**Review by toddlees — CHANGES_REQUESTED** [review]

File: tasks.py
Comment (lbeaufort) [reply tasks.py:89]: Thanks @toddlees! Fixed

**Review by toddlees — APPROVED** [review]

File: manifests/manifest-prod-web-services.yml
Comment (mjtravers) [inline manifests/manifest-prod-web-services.yml:15]: Can we add the creds service as well? The web service needs the FEC_FILING_API_KEY and we're currently storing it in both the web service app variables and the creds service. It'd be this addition:

      - fecfile-api-creds-prod

Similarly, dev and stage are fecfile-api-creds-dev and fecfile-api-creds-stage for those manifests.

Comment (lbeaufort) [reply manifests/manifest-prod-web-services.yml:15]: @mjtravers makes sense to me! Would you like the work to tell Celery to look at the creds service instead of its own env var to be its own ticket? We'll have to wait until the changes are merged/deployed in all the spaces, so it'll probably be a new sprint.

Comment (mjtravers) [reply manifests/manifest-prod-web-services.yml:15]: Yes, let's get the manifests updated in this ticket and we can rearrange the application variables in a later ticket after the updated manifests have worked their way through the 3 spaces.

Comment (lbeaufort) [reply manifests/manifest-prod-web-services.yml:15]: @mjtravers manifest changes made, follow-up ticket here: https://github.com/fecgov/fecfile-web-api/issues/644

**Review by mjtravers — APPROVED** [review]

**Pull request merged**
